.github/workflows/ci.yml

```diff
@@ -118,6 +121,33 @@ jobs:
           SENTRY_SYMBOLICATOR_TEST_AWS_SECRET_ACCESS_KEY: ${{ secrets.SENTRY_SYMBOLICATOR_TEST_AWS_SECRET_ACCESS_KEY }}
         run: pytest --ci -n12 -vv tests/integration
 
+  doc-comments:
+    name: Rust doc comments
+    runs-on: ubuntu-latest
+    env:
+      RUSTDOCFLAGS: -Dwarnings
```

Comment:
not sure we should be doing it this way, according to the comment on the relay PR from a rust dev.

Reply:
Because this environment variable is set for every invocation of this job and does not change I don't think there's anything wrong with using it this way.  Cargo sees exactly the same flags when compiling all artefacts and re-uses the cache.  I just pushed a noop change and the entire cache got reused.